_data/speakers.yml

```diff
@@ -139,7 +139,7 @@ david_townshend:
 
 kent_beck:
     name: Kent Beck
-    company: Independent
+    company: 
```

Comment:
Why tho?

Reply:
Because I thought it looked silly on the home page. I can put it back if you want?